meta-lmp-base/recipes-sota/aktualizr/aktualizr-fioefi-env-rollback.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Aktualizr configuration snippet to enable Foundries.IO UEFI Capsule updates"
+HOMEPAGE = "https://github.com/advancedtelematic/aktualizr"
+SECTION = "base"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
+
+inherit allarch
+
+SRC_URI = "file://sota-fioefi-env.toml"
+
+PV = "1.0"
```

Comment:
I prefer if we keep the package version in the filename

Reply:
Agree.